src/Server.php

```diff
@@ -144,6 +144,8 @@ class Server extends AbstractServer
      */
     protected $response;
 
+    protected Server\System $system;
```

Comment:
Is this always initialized? 🤔

Reply:
this needs:

- public modifier
- without type
- Add `@internal` doc to avoid abuse usage

see https://3v4l.org/HYEML

Reply:
@samsonasik  I don't think it is publicly used. There is a `getSystem()` method.

> Is this always initialized?

It is initialized in constructor.

Reply:
> Is this always initialized? 🤔

It is set in registerSystemMethods() which is called in the __construct()

Reply:
> this needs:
> 
>     * public modifier
> 
>     * without type
> 
>     * Add `@internal` doc to avoid abuse usage

Done

Reply:
@Xerkus that is the cost of dynamic property in the first place, so it needs to be public for BC reason imo, except, it will goes to major change.